### PR TITLE
make noop variant pkg_resources compatible

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Comes from https://gitter.im/conda-forge/conda-forge.github.io?at=6115db794bfde341d9592e4b

In short, the current `noop` variant does not install `dataclasses` because this is part of the stdlib as of py37. However, `pip check` and others fail in this scenario because `pkg_resources.get_distribution` will not find it (even if it's importable; I guess they don't check for the stdlib). This PR adds a shim setup.py that will create the distribution metadata so that `pkg_resources` sees _something_.

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
Thanks @jaimergp! Please consider adding yourself as a maintainer too ;-p
